net-mail/cyrus-imapd/cyrus-imapd-3.6.7.ebuild

```diff
@@ -0,0 +1,277 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Keep an eye on https://www.cyrusimap.org/imap/developer/compiling.html!
+inherit autotools eapi9-ver flag-o-matic pam ssl-cert
+
+DESCRIPTION="The Cyrus IMAP Server"
+HOMEPAGE="https://www.cyrusimap.org/"
+SRC_URI="https://github.com/cyrusimap/${PN}/releases/download/${P}/${P}.tar.gz"
+
+LICENSE="BSD-with-attribution GPL-2"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="
+	afs backup calalarm caps clamav http kerberos ldap
+	mysql nntp pam perl postgres replication
+	ssl static-libs tcpd test xapian
+"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="
+	afs? ( kerberos )
+	calalarm? ( http )
+	http? ( xapian )
+"
+
+COMMON_DEPEND="
+	dev-libs/libpcre:=
```

Comment:
for versions of 3.6.x >=3.6.4 the code supports pcre2 perhaps this should be updated to that depend (and the configure)